unreveal/models.py

```diff
@@ -20,6 +20,12 @@ class Comment(models.Model):
     email = models.CharField(max_length=100)
     comment = models.CharField(max_length=500)
 
+class Rating(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False, auto_created=True)
+    place_id = models.CharField(max_length=100)
+    email = models.CharField(max_length=100)
+    rating = models.DecimalField(max_digits = 5,decimal_places = 2)
```

Comment:
`max_digis` would allow the user to add a rating of `12345`? Or allow only till 5?
I think the `max_digits` should be 2, with 1 `decimal_place`, to allow only 1.0 or 4.5.
Also, if there is a way to do not allow numbers bigger than 5.0 it will be good.

Reply:
@dominikJakov ?

Reply:
So I put the max_digits to 2 and the decimal_place is 1.

Reply:
And is there any option to set the max quantity to 5.0?

Reply:
Okay, I think I finished.